### PR TITLE
Remove termion

### PR DESCRIPTION
**Pull request by jgrund**
Remove termion dep which is not compatible with Windows.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: liblustreapi-types/build.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 use std::path::PathBuf;
 
 fn main() {
-    if cfg!(target_os = "macos") {
+    if cfg!(target_os = "macos") || cfg!(target_os = "windows") {
```
Comment (ip1981) [inline liblustreapi-types/build.rs]: May I suggest building on Linux only? Lustre is linux-only, no?

Comment (jgrund) [reply liblustreapi-types/build.rs]: I'll update.

**Comment by mkpankov** [comment]
Looks like we need even more `cfg`

```
PS C:\Users\_\Documents\GitHub\integrated-manager-for-lustre> cargo check
   Compiling amq-protocol v3.1.0
   Compiling lapin v0.28.4
   Compiling liblustreapi-types v0.1.0 (C:\Users\_\Documents\GitHub\integrated-manager-for-lustre\liblustreapi-types)
    Checking iml-util v0.1.0 (C:\Users\_\Documents\GitHub\integrated-manager-for-lustre\iml-util)
    Checking iml-manager-cli v0.1.0 (C:\Users\_\Documents\GitHub\integrated-manager-for-lustre\iml-manager-cli)
    Checking iml-mailbox v0.1.0 (C:\Users\_\Documents\GitHub\integrated-manager-for-lustre\iml-mailbox)
error[E0433]: failed to resolve: could not find `unix` in `os`
  --> iml-util\util.rs:51:13
   |
51 |         os::unix::{io::FromRawFd, net::UnixListener as NetUnixListener},
   |             ^^^^ could not find `unix` in `os`

error[E0432]: unresolved import `tokio::net::UnixListener`
  --> iml-util\util.rs:56:28
   |
56 |         net::{TcpListener, UnixListener},
   |                            ^^^^^^^^^^^^
   |                            |
   |                            no `UnixListener` in `net`
   |                            help: a similar name exists in the module: `TcpListener`

error[E0433]: failed to resolve: use of undeclared type or module `NetUnixListener`
   --> iml-util\util.rs:107:29
    |
107 |         let addr = unsafe { NetUnixListener::from_raw_fd(3) };
    |                             ^^^^^^^^^^^^^^^ use of undeclared type or module `NetUnixListener`

warning: unused import: `convert::TryFrom`
  --> iml-util\util.rs:49:9
   |
49 |         convert::TryFrom,
   |         ^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

error: aborting due to 3 previous errors

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `iml-util`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by jgrund** [comment]
@mkpankov can the `iml-manager-cli` crate have `cargo check` run on it?

I'm not sure we can get all the way to the whole workspace as things like unix domain sockets don't have a clean analog on windows as far as I know.

**Pull request merged**